src/network_commands.rs

```diff
@@ -61,13 +61,12 @@ pub async fn perform_fixed_interval_network_churn(
 
     let nodes_per_vm = {
         let mut vms_to_ignore = 0;
-        inventory.vm_list.iter().for_each(|(name, _addr)| {
+        inventory.vm_list().iter().for_each(|(name, _addr)| {
             if name.contains("build") || name.contains("genesis") {
                 vms_to_ignore += 1;
```

Comment:
This is not related to this PR, but we should look into this command again to make sure that it is doing what is intended. The nodes_per_vm might be incorrect here and so will be the actual command.

Reply:
Same with the below command.

Reply:
Honestly, I don't know where these are used. I think they are in your nightly test runs?

Reply:
Yup, on my nightly tests which are again in a bad state rn. I'll take a look at it later then!

Reply:
On this one, I think I'll leave it for this PR. I don't know what modification I'd need to make.